src/pages/PrintPage.tsx

```diff
@@ -233,12 +239,14 @@ class SummaryPage extends React.Component<RouteComponentProps, State> {
         <Modal
           isOpen={this.state.showConfirmModal}
           centerContent
+          ariaLabel=""
```

Comment:
Why empty? Mistake?

Reply:
Not a mistake, meant to override the default label that is inside Modal. Again, trying to disrupt as little as possible.